Add tests/__init__.py to allow tests to import influxdb
--- /dev/null
+++ b/tests/__init__.py
@@ -0,0 +1 @@
+# added to fix tests
